cmd/manager/main.go

```diff
@@ -85,7 +88,18 @@ func initStaticDeps(mgr manager.Manager) {
 		glog.Fatalf("Could not create config watch: %v", err)
 	}
 
+	bootstrapKubeconfig, err := ioutil.ReadFile(*bootstrapKubeconfig)
```

Comment:
This effectively makes these flags required - is that deliberate?

Reply:
Nope! Thanks for catching.